README.md

```diff
@@ -6,3 +6,8 @@ at the moment. They might not be up to date.
 
 The branch names begin with a prefix indicating a package manager, followed by the
 test case name in the format: `<package-manager>/<test-case>`.
+
+To create a new test please create a branch in GitHub UI first and then open a PR
+against this branch. Alternatively start with pushing an empty branch first.
+Please avoid pushing non-empty branches since that would disrupt history of the
```

Comment:
How about instead of _since that would disrupt history of the project_ we'd say:

_since that would hinder the visibility of changes made to this repository in context of the main project repository._